### ISSUE TITLE
REST project unit tests don't work on Windows

### ISSUE THREAD

**Issue by DavidWhitlockTripwire** [DavidWhitlock/PortlandStateJava#303]
Looks like one of the recently-added unit tests doesn't work on Windows because a newline issues:

```
[INFO] Running edu.pdx.cs410J.whitlock.PhoneBillServletTest
[ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.056 s <<< FAILURE! - in edu.pdx.cs410J.whitlock.PhoneBillServletTest
[ERROR] addOneWordToDictionary(edu.pdx.cs410J.whitlock.PhoneBillServletTest)  Time elapsed: 1.839 s  <<< FAILURE!
java.lang.AssertionError:

Expected: "Defined TEST WORD as TEST DEFINITION\n"
     but: was "Defined TEST WORD as TEST DEFINITION\r\n"
        at edu.pdx.cs410J.whitlock.PhoneBillServletTest.addOneWordToDictionary(PhoneBillServletTest.java:61)
```

I should fix that.

**Comment by DavidWhitlock** [comment]
Also, the more modern `maven-jar-plugin` requires that the jar file has a "classifier" (I chose `client`), but this appears to break the shading of the jar file.  So, I can't create an uber jar in the same project as a war file.  That kind of makes sense.  In order to create an executable jar with the necessary dependencies, I had to create a `Class-Path` entry in the jar's manifest:
   * https://github.com/DavidWhitlock/PortlandStateJavaSummer2020/commit/dd416b0d3461d16b52768c854cd45646dad58137
  * https://github.com/DavidWhitlock/PortlandStateJavaSummer2020/commit/530bee56d6d159442668a56565457e78bbdf9164

All of this needs to get integrated into the web project archetypes.
